diff --git a/common/jpeg/bench/verilog/bench_top.v b/common/jpeg/bench/verilog/bench_top.v
index aed6314..f520d6d 100755
--- a/common/jpeg/bench/verilog/bench_top.v
+++ b/common/jpeg/bench/verilog/bench_top.v
@@ -596,12 +596,12 @@ module fdct_qnr_checker(clk, rst, dstrb, din);
 	  output_list[01] <= 11'h000;
 	  output_list[02] <= 11'h7fe;
 	  output_list[03] <= 11'h7ff;
-	  output_list[04] <= 11'h7ff;
+	  output_list[04] <= 11'h7fe;
 	  output_list[05] <= 11'h7ff;
 	  output_list[06] <= 11'h000;
 	  output_list[07] <= 11'h000;
 	  output_list[08] <= 11'h7ff;
-	  output_list[09] <= 11'h000;
+	  output_list[09] <= 11'h7ff;
 	  output_list[10] <= 11'h000;
 	  output_list[11] <= 11'h000;
 	  output_list[12] <= 11'h000;
@@ -798,7 +798,7 @@ module fdct_qnr_checker(clk, rst, dstrb, din);
 	  output_list[197] <= 11'h002;
 	  output_list[198] <= 11'h001;
 	  output_list[199] <= 11'h003;
-	  output_list[200] <= 11'h7fb;
+	  output_list[200] <= 11'h7fa;
 	  output_list[201] <= 11'h001;
 	  output_list[202] <= 11'h000;
 	  output_list[203] <= 11'h000;
