diff --git a/Pre-Synthesis_Simulation/test_tpu.v b/Pre-Synthesis_Simulation/test_tpu.v
index 7e5fdc8..ea95a42 100644
--- a/Pre-Synthesis_Simulation/test_tpu.v
+++ b/Pre-Synthesis_Simulation/test_tpu.v
@@ -198,8 +198,8 @@ sram_16x128b sram_16x128b_c2(
 
 //dump wave file
 initial begin
-  $fsdbDumpfile("tpu.fsdb"); // "gray.fsdb" can be replaced into any name you want
-  $fsdbDumpvars("+mda");              // but make sure in .fsdb format
+ // &
+ // &
 end
 
 //====== clock generation =====
@@ -242,11 +242,11 @@ initial begin
 end
 */
 initial begin
-    $readmemb("data/mat1.txt", mat1);
-    $readmemb("data/mat2.txt", mat2);
-    $readmemb("golden/golden1.txt",golden1);
-    $readmemb("golden/golden2.txt",golden2);
-    $readmemb("golden/golden3.txt",golden3);
+    $readmemb("mat1.txt", mat1);
+    $readmemb("mat2.txt", mat2);
+    $readmemb("golden1.txt",golden1);
+    $readmemb("golden2.txt",golden2);
+    $readmemb("golden3.txt",golden3);
 
     #(`cycle_period);
     
@@ -314,7 +314,7 @@ initial begin
 
 	end
       
-    $display("Total cycle count C after three matrix evaluation = %d.", cycle_cnt);
+    $display("TPU_ALL_PASSED"); $display("Total cycle count C after three matrix evaluation = %d.", cycle_cnt);
     #5 $finish;
 end
 
